schemas.py

```diff
@@ -0,0 +1,37 @@
+from datetime import date
+
+from pydantic import BaseModel
+
+
+class AuthorBase(BaseModel):
+    name: str
+    bio: str
+
+
+class AuthorCreate(AuthorBase):
+    pass
+
+
+class AuthorList(AuthorBase):
+    id: int
+
+    class Config:
+        orm_model = True
+
+
+class BookBase(BaseModel):
+    title: str
+    summary: str
+    publication_date: date
+
+
+class BookCreate(BookBase):
+    author_id: int
+
+
+class BookList(BookBase):
+    id: int
+    author: AuthorList
+
+    class Config:
+        orm_model = True
```

Comment:
what is `orm_model` for?
(https://docs.pydantic.dev/latest/migration/#changes-to-config)

The following config settings have been renamed:

orm_mode → from_attributes